lib/Raven/Client.php

```diff
@@ -1301,7 +1309,7 @@ protected function send_http_synchronous($url, $data, $headers)
 
         $errno = curl_errno($this->_curl_instance);
         // CURLE_SSL_CACERT || CURLE_SSL_CACERT_BADFILE
-        if ((($errno == 60) || ($errno == 77)) && !is_null($ca_cert)) {
+        if ((($errno == 60) || ($errno == 77) || ($errno == 28)) && !is_null($ca_cert)) {
```

Comment:
Maybe errno 28 could be seen only in my [testing http server](https://github.com/nokitakaze/php-testhttpserver) because I did not set the task for compatibility with [RFC 7230](http://httpwg.org/specs/rfc7230.html)